src/map/npc.c

```diff
@@ -3724,7 +3724,7 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 		return strchr(start,'\n');// skip and continue
```

Comment:
There is a function for checking the event name syntax. I was referring to that one. Of course we could add checking of the event labels after everything was fully booted up too.